pkg/tasks/probes.go

```diff
@@ -37,20 +37,44 @@ import (
 )
 
 const (
-	systemdShowStatusCMD = `systemctl show %s -p LoadState,ActiveState,SubState`
+	systemdShowStatusCMD    = `systemctl show %s -p LoadState,ActiveState,SubState`
+	systemdShowExecStartCMD = `systemctl show %s -p ExecStart`
 
-	dockerVersionDPKG   = `dpkg-query --show --showformat='${Version}' docker-ce | cut -d: -f2 | cut -d~ -f1`
-	dockerVersionRPM    = `rpm -qa --queryformat '%{RPMTAG_VERSION}' docker-ce`
-	dockerVersionAmazon = `rpm -qa --queryformat '%{RPMTAG_VERSION}' docker`
+	kubeletInitializedCMD = `test -f /etc/kubernetes/kubelet.conf`
+)
 
-	containerdVersion = `containerd --version | awk '{print $3}'`
+func safeguard(s *state.State) error {
```

Comment:
Should we allow users to skip this safeguard with the `--force` flag?

Reply:
Not now, but maybe in the future when possibly we will figure out the way to migrate between runtimes.